pkg/config/process_test.go

```diff
@@ -0,0 +1,79 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// allProcessSettings is a slice that contains details for testing regarding process agent config settings
```

Comment:
Correct me if I'm wrong but we're not testing the behavior of `DD_LOG_LEVEL` and `DD_LOG_TO_CONSOLE` here, right?

What do you think of adding a test for each setting like `TestXConfig` for `log_file`, `log_level`, `log_to_console`, `dd_agent_bin`, `grpc_client_timeout`, `remote_tagger`, `process_discovery` in the `pkg/process/config/config_test.go` file?

For each test we could ensure that:
* default value is respected
* we can override it with `DD_PROCESS_CONFIG_` or `DD_PROCESS_AGENT_`
* we can override it with `datadog.yaml` file.

@xornivore what do you think about it?

Reply:
Definitely recommend adding these even if they work as expected it provides additional regression coverage.

Reply:
Sure I can do that